BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoiceViewController.swift

```diff
@@ -102,6 +97,21 @@ public class DigitalInvoiceViewController: UIViewController {
         tableView.backgroundColor = UIColor.from(giniColor: returnAssistantConfiguration.digitalInvoiceBackgroundColor)
     }
     
+    fileprivate func configureNavigationBar() {
+        title = .ginibankLocalized(resource: DigitalInvoiceStrings.screenTitle)
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: prefferedImage(named: "infoIcon"), style: .plain, target: self, action: #selector(whatIsThisTapped(source:)))
+        let leftBarButtonItemTitle = String.ginibankLocalized(resource: DigitalInvoiceStrings.backButtonTitle)
+        navigationItem.leftBarButtonItem = GiniBarButtonItem.init(image: prefferedImage(named: "navigationDigitalInvoiceBack"), title: leftBarButtonItemTitle, style: .plain, target: self, action: #selector(closeReturnAssistantOverview))
```

Comment:
Could we reuse "arrowBack" for the back image instead of introducing a new image? Makes customisation easier.

Also the back button customisation should be added to the customisation guide.